includes/WPGraphQLContentBlocks.php

```diff
@@ -119,6 +110,38 @@ static function () {
 	 */
 	public function actions(): void {
 		add_action( 'graphql_register_types', [ $this, 'init_block_editor_registry' ] );
+		$this->register_wpgraphql_update_filters();
+	}
+
+	/**
+	 * Register filters related to WPGraphQL version compatibility.
+	 *
+	 * @since 0.0.1
+	 */
+	public function register_wpgraphql_update_filters(): void {
+		add_filter( 'graphql_get_dependents', [ $this, 'add_as_wpgraphql_dependent' ], 10, 2 );
```

Comment:
Since you're using the header, you dont need to force-add yourself using a filter.

Reply:
Does it work without the filter though? Just keeping the following lines in `wp-graphql-content-blocks.php`

```
 * Requires WPGraphQL:       1.14.5
 * WPGraphQL tested up to:   2.1.1
 ```
 should be enough?

Reply:
I mean I'm not sure if WordPress (what wpgraphql) uses underneath is able to handle the whitespace formatting, but yeah that's enough.